packages/launchpad-navigator/src/floating-navigator/index.tsx

```diff
@@ -8,9 +8,16 @@ export type FloatingNavigatorProps = {
 };
 
 const FloatingNavigator = ( { siteSlug }: FloatingNavigatorProps ) => {
+	const launchpadContext = 'navigator';
```

Comment:
It may be worth using `launchpad-navigator` as a more specific identifier.
```suggestion
	const launchpadContext = 'launchpad-navigator';
```